extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -248,9 +247,9 @@ private function clearUsers() {
 	 * @return string homepage URL. Default is US homepage.
 	 */
 	private function getHomepageUrl() {
-		global $wgLang;
+		global $wgContLang;
 		if ( class_exists('WikiaLogoHelper') ) {
-			return ( new WikiaLogoHelper() )->getCentralUrlForLang( $wgLang->getCode() );
+			return ( new WikiaLogoHelper() )->getCentralUrlForLang( $wgContLang->getCode() );
```

Comment:
It doesn't work anyway as it's cached for the first user visiting the wiki (for 60 seconds).